bundle/edu.gemini.pit/src/main/scala/edu/gemini/pit/ui/view/tac/TacView.scala

```diff
@@ -188,7 +189,7 @@ class TacView(loc: Locale) extends BorderPanel with BoundView[ProposalClass] { v
       override def refresh(m:Option[Option[SubmissionDecision]]) {
```

Comment:
I agree, that looks odd, I'll quickly review if it could/should be flattened up on the call chain